runtime/src/tiered_storage.rs

```diff
@@ -8,6 +10,65 @@ pub mod meta;
 pub mod mmap_utils;
 pub mod readable;
 
-use crate::tiered_storage::error::TieredStorageError;
+use {
+    error::TieredStorageError,
+    footer::{AccountBlockFormat, AccountMetaFormat, OwnersBlockFormat},
+    index::AccountIndexFormat,
+    std::path::{Path, PathBuf},
+};
```

Comment:
I was trying to follow the way we did under `src/runtime` where bigger modules have both .rs and a directory (like bank.rs and /bank, append_vec.rs and /append_vec).

And since this will be the main struct that interacts with the accounts db code, I am putting implementations here.

But let me know if you prefer to have everything under `/tiered_storage` instead.

Reply:
Renamed the struct to `TieredStorage`.  Hopefully it makes more sense to have it here.